### PR TITLE
[10.0] [MIG] web_timeline

### PR DESCRIPTION
**Pull request by LeartS**
Migration of `web_timeline` to v10.
See https://github.com/OCA/web/pull/393

### CODE REVIEW

**Comment by LeartS** [comment]
What's the problem with runbot?

File: web_timeline/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Web timeline",
+    'summary': "Interactive visualization chart to show events in time",
+    "version": "10.0.1.0.0",
+    'author': 'ACSONE SA/NV, '
+              'Tecnativa, '
+              'Monk Software, ',
```
Comment (alexis-via) [inline web_timeline/__manifest__.py]: You should remove the coma at the end of this line. It makes the module uninstallable.

File: web_timeline/README.rst [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+the value that represents the block. You can also click on the group name to
+edit the involved record directly.
+
+Double-click on the record to edit it. Double-click in open area to create a
+new record with the group and start date linked to the area you clicked in.
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/8.0
```
Comment (ThomasBinsfeld) [inline web_timeline/README.rst:96]: @LeartS s/8.0/10.0

**Review by lmignon — CHANGES_REQUESTED** [review]
The history from https://github.com/OCA/web/pull/521 is not preserved! 
see https://github.com/OCA/web/pull/306#issuecomment-292890328

File: web_timeline/__manifest__.py (hunk 1 above)
Comment (ThomasBinsfeld) [reply web_timeline/__manifest__.py]: Fixed with https://github.com/monksoftware/web/pull/2

Comment (lmignon) [reply web_timeline/__manifest__.py]: @LeartS @ThomasBinsfeld IMO The merged fix is not right!
Now the code looks like:
```python
'author': 'ACSONE SA/NV, '
          'Tecnativa, '
          'Monk Software, ',  # here the comma is not on the right line
          'Odoo Community Association (OCA)'
```
and should be
```python
'author': 'ACSONE SA/NV, '
          'Tecnativa, '
          'Monk Software, '
          'Odoo Community Association (OCA)',
```

**Review by lmignon — CHANGES_REQUESTED** [review]
@LeartS PLZ restore the commit history from #521 at least the date of the initial commit must be the same as in #521

**Comment by pedrobaeza** [comment]
@lmignon I haven't had time for restoring the commit history on v9, and as this PR is based on that, blame on me. I'll do it this week.

File: web_timeline/__manifest__.py (hunk 1 above)
Comment (ThomasBinsfeld) [reply web_timeline/__manifest__.py]: @lmignon Right, thanks. New PR fixes that https://github.com/monksoftware/web/pull/3

Comment (LeartS) [reply web_timeline/__manifest__.py]: Sorry, I merged without checking well 😞

**Comment by LeartS** [comment]
Ok, the commas should be fixed now. I'll wait for @pedrobaeza to restore the commit history on the v9 PR, and then rebase this.

**Comment by pedrobaeza** [comment]
I have fixed the commit history following v9 branch and now this is ready to be reviewed.

**Comment by pedrobaeza** [comment]
The ideal is to have the module project_timeline also migrated and point in the PR to this branch as I did for v9 in https://github.com/OCA/project/pull/187/commits/8016c54da8dc37a017d5fe8d753e9d424e9195ba

**Comment by pedrobaeza** [comment]
@LeartS can you please make a PR with project_timeline for checking the functionality as I pointed in my previous comment?

**Review by carlosdauden — APPROVED** [review]
Tested with https://github.com/OCA/project/pull/297
Works fine. :+1:

**Review by pedrobaeza — APPROVED** [review]

**Comment by LeartS** [comment]
Sorry guys, I am working on a lot of stuff at the moment (some of it not odoo related) and having trouble finding the time to follow my PRs and issues. Thanks @carlosdauden for the project PR and thanks everyone for the reviews and merge!

**Pull request merged**